core/src/mindustry/world/blocks/experimental/BlockForge.java

```diff
@@ -18,13 +19,16 @@
 import mindustry.world.blocks.production.*;
 import mindustry.world.consumers.*;
 
+import static mindustry.Vars.*;
+
 public class BlockForge extends PayloadAcceptor{
     public float buildSpeed = 0.4f;
+    public int minBlockSize = 1;
```

Comment:
also these two vars should be on the same line

Reply:
Done.